### PR TITLE
docs: replace template README and CONTRIBUTING with hpc-libs specific ones

### PR DESCRIPTION
**Pull request by NucciTheBoss**
This PR fleshes out the README and CONTRIBUTING documentation for `hpc-libs` so that it is inline with the other Charmed HPC repositories. This will help with discovery as well as informing people new to Charmed HPC to what hpc-libs actually is :shipit: 

_Who doesn't love badges and emojiis??_

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,34 +1,232 @@
...
+yourself with the [Juju SDK](https://juju.is/docs/sdk) as it will help you
+better understand how the Slurm charms are put together.
 
-```shell
-tox devenv -e integration
-source venv/bin/activate
-```
+Following these guidelines helps you communicate that you respect the maintainers
+managing the Slurm charms. In return, they will reciprocate that respect
+while addressing your issue or assessing your submitted changes and/or features.
```
Comment (jedel1043) [inline CONTRIBUTING.md:13]: `s/Slurm charms/hpc-libs charm libraries`?

Comment (NucciTheBoss) [reply CONTRIBUTING.md:13]: image

Comment (NucciTheBoss) [reply CONTRIBUTING.md]: _Fixed_ :japanese_ogre:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,34 +1,232 @@
...
+* `Type: Refactor` - Issues that pertain to improving the existing code base.
+
+* `Type: Idea Bank` - Issues that pertain to proposing potential improvements to the code base.
+
+* `Type: Enhancement` - Issues marked as an agreed upon enhancement to the code base. Can also be used for pull requests.
+
+* `Status: Help wanted` - Issues where we need help from the greater open source community to solve.
+
+For a complete look at this repository's labels, see the
+[project labels page](https://github.com/charmed-hpc/slurm-charms/labels).
```
Comment (jedel1043) [inline CONTRIBUTING.md:72]: Heh, got another one

Comment (NucciTheBoss) [reply CONTRIBUTING.md]: Rats, good catch. Need to up that Ctrl+F game :sob:

File: README.md [hunk 3]
```diff
@@ -1,26 +1,74 @@
...
+* [`is_container` documentation](https://charmhub.io/hpc-libs/libraries/is_container)
+* [`slurm_ops` documentation](https://charmhub.io/hpc-libs/libraries/slurm_ops)
+
+## 🤔 What's next?
+
+If you want to learn more about all the things you can do with the hpc-libs charm libraries,
+or have any further questions on what you can do with the charm libraries, here are some
+further resources for you to explore:
+
+* [Open an issue](https://github.com/charmed-hpc/slurmutils/issues/new?title=ISSUE+TITLE&body=*Please+describe+your+issue*)
```
Comment (jedel1043) [inline README.md:32]:
```suggestion
* [Open an issue](https://github.com/charmed-hpc/hpc-libs/issues/new?title=ISSUE+TITLE&body=*Please+describe+your+issue*)
```

Comment (NucciTheBoss) [reply README.md]: :clown_face: alright, we should be good now

**Review by jedel1043 — APPROVED** [review]
Fourth time was the charm! (for now...)

**Comment by NucciTheBoss** [comment]
> Fourth time was the charm! (for now...)

:mild_panic_intensifies:

**Pull request merged**